src/app/features/container/containers/popup/popup.layout.tsx

```diff
@@ -0,0 +1,106 @@
+import { useState } from 'react';
+import { Outlet, useLocation } from 'react-router-dom';
+
+import { ChainID } from '@stacks/transactions';
+import { Box } from 'leather-styles/jsx';
+
+import { Flag, Logo, NetworkModeBadge } from '@leather.io/ui';
+
+import { RouteUrls } from '@shared/route-urls';
+
+import { CurrentAccountAvatar } from '@app/features/current-account/current-account-avatar';
+import { CurrentAccountName } from '@app/features/current-account/current-account-name';
+import { SwitchAccountDialog } from '@app/features/dialogs/switch-account-dialog/switch-account-dialog';
+import { useCurrentNetworkState } from '@app/store/networks/networks.hooks';
+
+import { TotalBalance } from '../../total-balance';
+import { ContainerLayout } from '../container.layout';
+import { PopupHeader } from './popup-header';
+
```

Comment:
I left these localised switches here as we will be working on the popups in approval UX (`popup.layout.tsx` can become `approval.layout`). They toggle what is and isn't shown in the header. 

I can refactor to setup a specific context for this if needed